l3kernel/testfiles/m3bitset001.tlg

```diff
@@ -1,2605 +1,194 @@
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\OT1/cmr/m/n/10 0
-....\penalty 10000
-....\glue(\parfillskip) 0.0 plus 1.0fil
-....\glue(\rightskip) 0.0
-...\glue 0.0 plus 1.0fil
-...\glue 0.0
-...\glue 0.0 plus 0.0001fil
-..\glue(\baselineskip) 23.55556
-..\hbox(6.44444+0.0)x345.0, glue set 170.0fil
-...\glue 0.0 plus 1.0fil
-...\OT1/cmr/m/n/10 2
-...\glue 0.0 plus 1.0fil
-(m3bitset001.aux)
+l. ... }
+============================================================
+============================================================
+TEST 8: show small-index
+============================================================
+101010101
+0
+0
+1
+============================================================
+============================================================
+TEST 9: show small-index-global
+============================================================
+101010101
+0
+0
+1
+============================================================
+============================================================
+TEST 10: show export
+============================================================
+101010101
+341
+============================================================
+============================================================
+TEST 11: show export global
+============================================================
+101010101
+341
+============================================================
+============================================================
+TEST 12: show large index
+============================================================
+10000000000000000000000000000000101010101
+0
+1
+0
+============================================================
+============================================================
+TEST 13: show large index global
+============================================================
+10000000000000000000000000000000101010101
+0
+1
+0
+============================================================
+============================================================
+TEST 14: show index 8
+============================================================
+0
+============================================================
+============================================================
+TEST 15: show index 8 global
+============================================================
+0
+============================================================
+============================================================
+TEST 16: add index 8 with sum
+============================================================
+! Arithmetic overflow.
```

Comment:
Are these overflows deliberate?

Reply:
yes, at least I didn't overlooked them. It happens as `\bitset_to_arabic:N` can't handle larger bitset (yet). As this would change if the internal implemention is switched to something else, I thought having a test which would react to such a change would be sensible.